frontend/server/src/main/java/com/amazonaws/ml/mms/util/ConfigManager.java

```diff
@@ -431,6 +441,14 @@ private int getIntProperty(String key, int def) {
         return Integer.parseInt(value);
     }
 
+    public String getDefaultResponseTimeout() {
```

Comment:
It's better return integer.

Reply:
The way I have the code is, we consume this as String in the beginning, but convert to Integer only once per model loading. Would this cause any issues?